core/src/main/java/com/netflix/iceberg/util/BinPacking.java

```diff
@@ -33,20 +34,22 @@
   public static class ListPacker<T> {
     private final long targetWeight;
     private final int lookback;
+    private final boolean expensiveTaskFirst;
 
-    public ListPacker(long targetWeight, int lookback) {
+    public ListPacker(long targetWeight, int lookback, boolean expensiveTaskFirst) {
```

Comment:
Does `ListPacker` need to support this option?

Reply:
@rdblue Arguably no. But since it does support an API to instantiate a `PackingIterable`, I thought to keep the semantics same and allow argument to be passed via `ListPacker`.